Formula/cpr.rb

```diff
@@ -44,6 +46,7 @@ def install
   test do
     (testpath/"test.cpp").write <<~EOS
       #include <iostream>
+      #include <curl/curl.h>
```

Comment:
Is this line needed though? Since there's no `curl` code in the program.

Reply:
No, probably not. But `cpr/cpr.h` does seem to include it transitively.